requirements.txt

```diff
@@ -1,15 +1,19 @@
+astroid==2.3.3
 certifi==2020.4.5.1
 chardet==3.0.4
-future==0.18.2
 idna==2.9
 influxdb==5.3.0
-iso8601==0.1.12
+isort==4.3.21
+lazy-object-proxy==1.4.3
+mccabe==0.6.1
```

Comment:
There are quite some packages in here that aren't needed for the actual code as far as I can tell. Where did those come from?

Reply:
no idea. Can be removed